.github/workflows/e2e-tests.yml

```diff
@@ -167,4 +167,33 @@ jobs:
           event_type: e2e
           report_name: ${{needs.create-test-matrix.outputs.runName}}
 
+  slack-notification:
+    name: "Slack notification"
+    runs-on: ubuntu-latest
+    if: ${{ ! cancelled() }}
+    needs: [e2e-tests]
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up tools
+        uses: ./.github/actions/tool-setup
+
+      # remove this step after action release and replace with the action from marketplace
```

Comment:
I don't think that's necessary, we can keep using the action from inside the monorepo in the future; it will make future PRs easier too.

```suggestion
```